cmd/main.go

```diff
@@ -95,13 +94,10 @@ func main() {
 	memcache := &server.Memcache{}
 	if *enableBranchCache {
 		// Memcache
-		branchCacheFolder := *branchFolder
-		if branchCacheFolder == "" {
-			branchCacheFolder, err = server.ReadBranchCacheFolder(
-				ctx, branchCacheBucket, branchCacheVersionFile)
-			if err != nil {
-				log.Fatalf("Failed to read branch cache folder: %v", err)
-			}
+		branchCacheFolder, err := server.ReadBranchCacheFolder(
+			ctx, branchCacheBucket, branchCacheVersionFile)
+		if err != nil {
+			log.Fatalf("Failed to read branch cache folder: %v", err)
```

Comment:
Should this be fatal or more graceful?

Reply:
there is no monitoring on errors yet, for now can make it fatal so we know the branch cache load has issue?

Reply:
ok. and if bad things happen we just set enableBranchCache to false...

Reply:
sg